python/cudf/cudf/testing/_utils.py

```diff
@@ -306,7 +311,7 @@ def gen_rand(dtype, size, **kwargs):
             np.random.randint(low=low, high=high, size=size), unit=time_unit
         )
     elif dtype.kind in ("O", "U"):
-        return pd.util.testing.rands_array(10, size)
+        return pd._testing.rands_array(10, size)
```

Comment:
I would prefer to avoid relying on pandas' non-public testing functions, but it seems like the previous name no longer exists. This is probably our best choice for now unless we want to re-implement it.

Reply:
Yeah, it isn't ideal, but it is what we have right now. Should we make an issue for it?

Reply:
I searched the code base for `pd._testing` and I see a lot of results. This is not an isolated occurrence, which either means "we accept this risk of depending on pandas internals" or "we need to address this problem systemically." An issue would be good.

Reply:
Yeah I had the same concern when Mike showed me the solution, but there isn't an obvious replacement. I would say replacing it can be the job of a full-time Python dev in the future.

Reply:
https://github.com/rapidsai/cudf/issues/10777